modules/microshift-restart-ovnkube-master.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
```

Comment:
Is this restarting of ovnkube-master pod only referenced by mtu change or is it also referenced by the firewall known issue?
If it is for mtu, then we probably don't need it since node reboot will restart ovnkube-master.

Reply:
@zshi-redhat OK, great, thank you. That was the question I had asked above (do we need to do both).